src/libraries/System.Formats.Cbor/src/System/Formats/Cbor/CborConformanceLevel.cs

```diff
@@ -17,9 +17,9 @@ public enum CborConformanceMode
         /// <para>Ensures that the CBOR data adheres to strict mode, as specified in RFC7049 section 3.10.</para>
         /// <para>Extends lax conformance with the following requirements:</para>
         /// <list type="bullet">
-        /// <item>Maps (major type 5) must not contain duplicate keys.</item>
-        /// <item>Simple values (major type 7) must be encoded as small a possible and exclude the reserved values 24-31.</item>
-        /// <item>UTF-8 string encodings must be valid.</item>
+        /// <item><description>Maps (major type 5) must not contain duplicate keys.</description></item>
```

Comment:
Why is `<description>` needed inside `<item>`?

Reply:
If you don't have `<description>`, you just see an empty bulleted list:

![image](https://user-images.githubusercontent.com/24882762/156699695-17dcf6e3-5818-44e7-b3c9-bddc048ef810.png)

Reply:
That's described in https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments#d37-list